src/common/Button/Button.tsx

```diff
@@ -0,0 +1,13 @@
+import * as types from 'src/types';
```

Comment:
For the next PR: A type for a component props is rarely used in other parts of an application but in the component file itself (and test file for this component), so it's ok if you put `ButtonProps` in this file, but keep global types like `Course` or `User` in `types.ts`